packages/neuron-wallet/src/utils/logger.ts

```diff
@@ -10,13 +15,18 @@ const logger = winston.createLogger({
   format: combine(timestamp(), json()),
   transports: [
     new winston.transports.File({
-      filename: 'combined.log',
+      filename: `${path}combined.log`,
```

Comment:
Question: doesn't app usually write log files to platform's default system log locations?

Reply:
This PR is closed for a (new one](https://github.com/nervosnetwork/neuron/pull/249) which saves logs to `app.getPath('logs')`